src/lib/optimizer/join_ordering/abstract_join_ordering_algorithm.cpp

```diff
@@ -58,8 +58,13 @@ std::shared_ptr<AbstractLQPNode> AbstractJoinOrderingAlgorithm::_add_join_to_pla
   }
 
   /**
-   * Join two plans using a set of predicates; try to bring them into an efficient order
+   * Join two plans using a set of predicates; try to bring them into an efficient order.
```

Comment:
Since `them` could also refer to `two plans`, maybe change it to `try to bring the predicates into an efficient order`

Reply:
Yepp. Also changed "plans" to "subplans"